modules/react-native-chat/package.json

```diff
@@ -1,15 +1,34 @@
 {
   "name": "@modules/chat",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "description": "A set of screens to display chat conversations in a list and 1-to-1 messaging",
   "private": true,
   "main": "index.js",
   "author": "Crowdbotics",
   "license": "ISC",
   "dependencies": {
-    "pubnub": "^4.29.11",
+    "pubnub": "^4.33.0",
+    "zustand": "^3.5.13",
     "pubnub-react": "^2.1.0",
-    "react-native-emoji-selector": "^0.2.0",
-    "react-native-keyboard-aware-scroll-view": "^0.9.3"
+    "is-url": "^1.2.4",
+    "react-native-gifted-chat": "^0.16.3",
+    "uuid": "^8.3.2",
+    "react-native-image-crop-picker": "^0.36.2",
+    "react-native-permissions": "^3.0.5",
+    "react-native-linear-gradient": "^2.4.0",
+    "react-native-image-picker": "4.7.3",
+    "@react-native-community/checkbox": "0.5.10",
+    "react-native-popup-menu": "0.15.12",
+    "react-native-video": "5.2.0",
+    "react-native-fs": "2.19.0",
+    "react-native-emoji-selector": "0.2.0"
+  },
+  "x-dependencies": {
+    "react-native-image-crop-picker": "0.36.2",
+    "react-native-permissions": "3.0.4",
+    "react-native-linear-gradient": "^2.4.0",
+    "react-native-image-picker": "4.7.3",
+    "@react-native-community/checkbox": "0.5.10",
+    "react-native-fs": "2.19.0"
```

Comment:
Add dependencies only in one place, please. If it depends on native code (react-native link) then it must go on the `x-dependencies` section.

Reply:
Resolved.